src/test/resources/epub3/mediaoverlays-package-document.feature

```diff
@@ -43,7 +43,14 @@ Feature: EPUB 3 ▸ Media Overlays ▸ Package Document Checks
   Scenario: the 'media:duration' property can be expressed as a full clock value
     When checking file 'mediaoverlays-duration-fullclock-valid.opf'
     Then no errors or warnings are reported
-    
+  
   Scenario: the 'media:duration' property can be expressed as a timecount value
     When checking file 'mediaoverlays-duration-timecount-valid.opf'
-    Then no errors or warnings are reported
\ No newline at end of file
+    Then no errors or warnings are reported
+  
+  Scenario: Repoort 'media:duration' properties with non-clock values
+    When checking file 'mediaoverlays-duration-clock-values-error.opf'
+    Then error RSC-005 is reported 3 times
+    And the message contains "must be a valid SMIL3 clock value"
```

Comment:
note that this message assertion will only apply to the last RSC-005 error. If you want to enforce it to all 3, you need to use the table syntax (see step `the following errors are reported:`).